index.html

```diff
@@ -802,10 +802,13 @@ <h3>Presentations</h3>
 A <a>verifiable presentation</a> expresses data from one or more
 <a>verifiable credentials</a>, and is packaged in such a way that the
 authorship of the data is <a>verifiable</a>. If <a>verifiable credentials</a>
-are presented directly, they become <a>verifiable presentations</a>. Data
-formats derived from <a>verifiable credentials</a> that are cryptographically
-<a>verifiable</a>, but do not of themselves contain
-<a>verifiable credentials</a>, might also be <a>verifiable presentations</a>.
+are presented directly, they become <a>verifiable presentations</a>.
+        </p>
+        <p>
+To support selective disclosure capabilities, data formats derived from
+<a>verifiable credentials</a>, that are cryptographically <a>verifiable</a> but
+are not themselves <a>verifiable credentials</a>, might also be
+<a>verifiable presentations</a>. 
```

Comment:
That also interferes with 6.10 Presentations.

Reply:
I think you meant 4.10 here. Can you point out more explicitly what contradiction you see?

Reply:
My apologies, I meant 4.10, correct.

The problem or confusion is the following. According to 4.10, a presentation typically has the following members:

> type
The type [property](https://w3c.github.io/vc-data-model/#dfn-property) is required and expresses the type of [presentation](https://w3c.github.io/vc-data-model/#dfn-presentation), such as VerifiablePresentation. For details related to the use of this property, see Section [4.4 Types](https://w3c.github.io/vc-data-model/#types).
>
> verifiableCredential
If present, the value of the verifiableCredential [property](https://w3c.github.io/vc-data-model/#dfn-property) MUST be constructed from one or more [verifiable credentials](https://w3c.github.io/vc-data-model/#dfn-verifiable-credential), or of data derived from [verifiable credentials](https://w3c.github.io/vc-data-model/#dfn-verifiable-credential) in a cryptographically [verifiable](https://w3c.github.io/vc-data-model/#dfn-verify) format.

I know that neither `type` nor `verifiableCredential` are mandatory as per VCDM.  But if VCs and derived VCs are verifiable presentations, then the definitions above lead to confusion. The definitions above suggest that VCs and derived VCs would be in the `verifiableCredential` array. If this is no longer the case and a derived VC is itself a VP, then we suddenly have two ways to encode a derived VC (or plain VC) in a VP.

Reply:
In general, I'm not sure if we want to have that polymorphism where we would allow VCs and derived VCs to be VPs which could lead to some interesting side effects and potentially semantic confusion where suddenly verifiers cannot disambiguate any longer whether fields such as `id`, `proof`, `type` etc. are VC metadata or related to the presentation.

Reply:
I find what @awoie is saying to make a lot of sense. It also fits in well with the transformation concept we have expressed elsewhere. My understanding is that this text was originally meant to help cover some bases around, for example, CL-protected / AnonCred style presentations, but based on recent demonstrations, I'm pretty sure it's no longer needed.

We should probably simplify the language to just talk about VPs (as they appear in the spec itself) and simplify the `verifiableCredential` property to just talk about VCs instead of "data derived from" them. People should be able to transform to / from VCs as needed or they should use another property in an extended VP to express other formats. It has been suggested elsewhere that we could add one or more other properties for other data to travel in VPs (such as VCs encrypted as JWEs); that seems the right way to go to avoid confusion.

Reply:
We should probably strike this whole paragraph as well. I don't think it means anything to say some undefined thing "might be a verifiable presentation". Not saying it doesn't prohibit anyone from doing selective disclosure or using other formats to present or transfer information that are derived from VCs that aren't VPs. Those other formats aren't VPs unless they comply with the data model requirements for VPs -- that's what interop is about. I think that's fine; either you can transform from those other things to VPs and present that way or you can't.

Reply:
I suggest removing the text, if it does not make sense and lacks consensus.